src/generic_chunk.c

```diff
@@ -69,13 +69,15 @@ ChunkResult handleDuplicateSample(DuplicatePolicy policy, Sample oldSample, Samp
         case DP_LAST:
             return CR_OK;
         case DP_MIN:
-            newSample->value = min(oldSample.value, newSample->value);
+            if (oldSample.value < newSample->value)
+                newSample->value = oldSample.value;
```

Comment:
in the past we had a max/min macro that did exactly this. but this is good enough as well.

Reply:
@danni-m did not used the macro to save unnecessary work when the conditions are not met ( meaning newSample does not need to be updated right ).